### PR TITLE
matched EnGeldB_DodgeRanged

### PR DESCRIPTION
**Pull request by engineer124**

### CODE REVIEW

File: src/overlays/actors/ovl_En_GeldB/z_en_geldb.c [hunk 1]
```diff
@@ -1534,12 +1532,12 @@ s32 EnGeldB_DodgeRanged(GlobalContext* globalCtx, EnGeldB* this) {
...
 
         angleToFacing = func_8002DA78(&this->actor, actor) - this->actor.shape.rot.y;
-        this->actor.posRot.rot.y = this->actor.shape.rot.y;
+        this->actor.posRot.rot.y = (u16) this->actor.shape.rot.y & 0xFFFF;
         dist = func_8002DB6C(&this->actor, &actor->posRot.pos);
         //! @bug
         // func_8002DB6C already sqrtfs the distance, so this actually checks for a
         // distance of 360000. Also it's a double calculation because no f on sqrt.
-        if ((ABS(angleToFacing) < 0x2EE0) && (sqrt(dist) < 600.0)) {
+        if ((ABS(angleToFacing) < 12000) && (sqrt(dist) < 600.0)) {
```
Comment (fig02) [inline src/overlays/actors/ovl_En_GeldB/z_en_geldb.c:1540]: we dont really have set rules on hex vs dec for angles at the moment, but if the rest of the file has them as hex you should prob keep it as hex

File: src/overlays/actors/ovl_En_GeldB/z_en_geldb.c [hunk 2]
```diff
@@ -1534,12 +1532,12 @@ s32 EnGeldB_DodgeRanged(GlobalContext* globalCtx, EnGeldB* this) {
         f32 dist;
 
         angleToFacing = func_8002DA78(&this->actor, actor) - this->actor.shape.rot.y;
-        this->actor.posRot.rot.y = this->actor.shape.rot.y;
+        this->actor.posRot.rot.y = (u16) this->actor.shape.rot.y & 0xFFFF;
```
Comment (fig02) [inline src/overlays/actors/ovl_En_GeldB/z_en_geldb.c:1535]: can remove the space between the cast and the var

**Comment by petrie911** [comment]
While you're on this file, could you change `(0, ` to `((void) 0,` earlier in the function to suppress a compiler warning?

**Comment by engineer124** [comment]
Sounds good Petrie

**Review by fig02 — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

**Pull request merged**